### DISCUSSION TITLE
copilot not activating

### DISCUSSION THREAD

**Discussion by codermaisha** [community/community#62515] [GitHub Education]
### Select Topic Area

Question

### Body

hello. i am new at github. and i applied for github copilot student package with my student id. but it showed my account is new so apply after some days so i waited and now again applying for it. but it shows now that my document has already been used. i have only one id card . so what to do right now. why i  cant apply with my id proof

**Comment by samus-aran** [comment]
Welcome to the GitHub Community, @coder-maisha , we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you.

**Comment by samus-aran** [comment]
Hi @coder-maisha 

That rejection reason is asking you to resubmit another instance of that piece of evidence. If you have taken a photo of that evidence then please take another rather than re-using the same photo. Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

Hope that helps 👍

**Reply by codermaisha** [reply]
i get unsuccessful using my student id. i am now at class 11. how could it be possible for me to find more evidence. as per github requirement i submitted my data under student id section .and it is rejecting me. please help me so solve this problem.

**Comment by Awrsha** [comment]
It sounds like there was a mistake or miscommunication with GitHub Copilot's application process. Here are some recommendations:

Double check that you are using the same email address and GitHub account that you used for your initial application. Make sure those match what is on your student ID.

Reach out directly to GitHub Copilot support for assistance. They will have the most insight into what happened with your specific application and can clarify the issue. You can contact them here: https://copilot.github.com/support/

Explain the situation clearly and politely to GitHub Copilot support - mention that you have only one student ID, you waited the required time before reapplying, but the system is still showing your document as already used.

Ask GitHub Copilot support for next steps to resolve the issue. They may ask you to resubmit your student ID, provide additional information, or make some other change to get your application approved.

Be patient - it may take a bit of time for GitHub Copilot support to review your case and respond. But working directly with them is the best chance of resolving the mix-up and getting your student package approved.

**Reply by codermaisha** [reply]
hi Awrsha. thank you for your response. the link you give me to contact the support is not working. it is showing page not found. i literally have no evidence rather than my student id. give me more tips how to resolve this issue
